README.md

```diff
@@ -208,3 +210,4 @@ Adding a new language:
 [useTranslation hook]: https://react.i18next.com/latest/usetranslation-hook
 [English translation.json]: src/locales/en/translation.json
 [i18n.ts]: src/locales/i18n.ts
+[language-icon-copyright]: https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-theme-classic/src/theme/Icon/Language/index.tsx
```

Comment:
I would move this link next to icon code, and make the link permanent.

We also need to copy their license, so restructure:
- src/styles/theme/icons/Language/index.tsx
- src/styles/theme/icons/Language/LICENSE

Reply:
Shouldn't all copyright stuff always go in the root LICENSE (or README) in that regard?

Reply:
Seems unusual. Usually this would be an npm package, and have node_modules/dep/LICENSE. So we did that for https://github.com/oasisprotocol/oasis-wallet-web/blob/master/src/fonts/rubik/LICENCE too :shrug: 

We probably should generate a page with all licenses combined tho https://github.com/oasisprotocol/oasis-wallet-web/issues/870

Reply:
Done.